### ISSUE TITLE
Add option in siac to display host contracts.

### ISSUE THREAD

**Issue by DavidVorick** [NebulousLabs/Sia#2465]
I'm not sure if this is even supported in the API yet, we may need to extend the API. But we need to be able to see all of the contracts that a host is currently using, which will help with debugging and also just help users be more generally aware of the status of their hosts.

**Comment by hudaniel** [comment]
Are there any dev environments we can use to test a wallet that accepted contracts?

**Comment by lukechampine** [comment]
@hudaniel you can setup a local testnet with https://github.com/NebulousLabs/Sia-Ant-Farm, though it isn't super well-documented

**Comment by hudaniel** [comment]
I'm getting this error when trying to `make dependencies`
```
$ go install -tags='dev' github.com/NebulousLabs/Sia/siad
can't load package: package github.com/NebulousLabs/Sia/siad: cannot find package "github.com/NebulousLabs/Sia/siad" in any of:
    /usr/local/Cellar/go/1.9.2/libexec/src/github.com/NebulousLabs/Sia/siad (from $GOROOT)
    /Users/Daniel/code/golang/src/github.com/NebulousLabs/Sia/siad (from $GOPATH)
``` 
any ideas?

**Comment by lukechampine** [comment]
hmm, `go install -tags='dev' github.com/NebulousLabs/Sia/siad` is the wrong command. The `siad` package has been moved to the `cmd` subfolder. I've pushed the fix.

**Comment by S-anasol** [comment]
In case someone want to imlement it:
https://github.com/S-anasol/Sia/blob/obligationsapi/api/host.go#L383-L387
Very simple api method to retrieve host contracts data.

UI for json exported with this api call:
https://github.com/S-anasol/hostdbviewer

UI Example/Tester/Viewer:
https://s-anasol.github.io/hostdbviewer/#/

**Comment by tbenz9** [comment]
@nielscastien are you planning on adding your new API endpoints into a `siac` command?

**Comment by nielscastien** [comment]
@tbenz9 No, but if one of the devs (or you?) can point me in the right direction on how to present and/or filter the data, I am willing to give it a try. There are quite some fields in the contract so I'm afraid that showing all will not work for the end user.

**Comment by nielscastien** [comment]
Something like `siac host contracts -s` (`-s` flag for sorting on expiration height):

![image](https://user-images.githubusercontent.com/22688069/38930361-f6b87fd0-430f-11e8-9651-df6de1cff27e.png)

**Comment by nielscastien** [comment]
or including locked and risked collateral:

![image](https://user-images.githubusercontent.com/22688069/38931269-d9b97274-4312-11e8-963a-8f7ddf99f213.png)

**Comment by nielscastien** [comment]
Another example:

![image](https://user-images.githubusercontent.com/22688069/38946552-14815a12-433a-11e8-8324-28bac3d136e2.png)

**Comment by tbenz9** [comment]
@DavidVorick can you specify which fields you think would be most useful in the `siac` output?  Looks like nielcastien has a great start, just needs a little clarification.

Available fields per contract
```{
  "contracts": [
    {
      "contractcost":			"1234",		// hastings
      "datasize":			500000,		// bytes
      "lockedcollateral":		"1234",		// hastings
      "obligationid":			"fff48010dcbbd6ba7ffd41bc4b25a3634ee58bbf688d2f06b7d5a0c837304e13",
      "potentialdownloadrevenue":	"1234",		// hastings
      "potentialstoragerevenue":	"1234",		// hastings
      "potentialuploadrevenue":		"1234",		// hastings
      "riskedcollateral":		"1234",		// hastings
      "sectorrootscount":		2,
      "transactionfeesadded":		"1234",		// hastings

      "expirationheight":		123456,		// blocks
      "negotiationheight":		123456,		// blocks
      "proofdeadline":			123456,		// blocks

      "obligationstatus":		"obligationFailed",
      "originconfirmed":		true,
      "proofconfirmed":			true,
      "proofconstructed":		true
      "revisionconfirmed":		false,
      "revisionconstructed":		false,
    }
  ]
}```

**Comment by DavidVorick** [comment]
I think the following would be most interesting to a user:

+ contract cost
+ total expected revenue
+ total money locked up
+ total amount of burn
+ expiration height

------

As a dev, I'm most curious about: the obligation status group, the whole group.
